tests/integration/utils.py

```diff
@@ -0,0 +1,84 @@
+import os
+from datetime import datetime
+from typing import Dict, List
+
+from confluent_kafka import Producer
+from confluent_kafka.admin import AdminClient, NewTopic
+
+
+def get_file_lines(filename: str) -> List:
+    with open(f'{os.path.dirname(__file__)}/resources/{filename}') as f_lines:
+        return f_lines.readlines()
+
+    return []
```

Comment:
This line never will be run, so better to remove it. If file is empty then `f_lines.readlines()` will be `[]` itself and if file not exist then raises an exception!